examples/tutorials/advanced/websockets-example-MNIST/run_websocket_client.py

```diff
@@ -68,7 +67,7 @@ def train_on_batches(worker, batches, model_in, device, lr):
         if batch_idx % LOG_INTERVAL == 0:
             loss = loss.get()  # <-- NEW: get the loss back
             loss_local = True
-            logger.debug(
+            print(
```

Comment:
Why not keeping logger.debug?

Reply:
logger was not printing anything on the notebook output, I don't know what was causing that maybe it was because we were getting functions from script instead of running script directly from terminal so I tried print and loss after LOG_INTERVAL start showing in notebook

Reply:
but I will try to  do few more test on logging

Reply:
logging working now had to add steam handlers